cpp/src/slice2cs/Gen.cpp

```diff
@@ -1820,7 +1820,8 @@ Slice::Gen::Gen(const string& base, const vector<string>& includePaths, const st
 
     printGeneratedHeader(_out, fileBase + ".ice");
 
-    _out << nl << "using _System = global::System;";
+    _out << nl << "#nullable enable";
+    _out << sp << nl << "using _System = global::System;";
```

Comment:
Seems we don't use `_System` anywhere, is this required?

Reply:
Yes, we should remove it. It's not used anywhere.